src/goto-checker/bmc_util.h

```diff
@@ -70,12 +70,26 @@ void slice(
   ui_message_handlert &);
 
 /// Sets property status to PASS for properties whose
-/// conditions are constant true.
-/// \return IDs of updated properties
-std::vector<irep_idt> update_properties_status_from_symex_target_equation(
+/// conditions are constant true in the \p equation.
+/// \param [inout] properties: The status is updated in this data structure
+/// \param [inout] updated_properties: The set of property IDs of
+///   updated properties
+/// \param equation: A equation generated by goto-symex
+void update_properties_status_from_symex_target_equation(
   propertiest &properties,
+  std::unordered_set<irep_idt> &updated_properties,
   const symex_target_equationt &equation);
 
+/// Sets property status for NOT_CHECKED properties.
```

Comment:
What will it set them to? Under what circumstances? Is this a subset of the changes made by the function above?